post_installation_configuration/configuring-multi-arch-compute-machines/multiarch-tuning-operator.adoc

```diff
@@ -6,30 +6,30 @@ include::_attributes/common-attributes.adoc[]
-For more information about the support scope of Red{nbsp}Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
-====
+The Multiarch Tuning Operator optimizes workload management within multi-architecture clusters and in single-architecture clusters transitioning to multi-architecture environments.
 
-The Multiarch Tuning Operator enhances the operational experience within multi-architecture clusters, and single-architecture clusters that are migrating to a multi-architecture compute configuration.
+Architecture-aware workload scheduling allows the scheduler to place pods onto nodes that match the architecture of the pod's images. 
 
-This Operator implements the `clusterpodplacementconfigs` custom resource (CR) to support architecture-aware workload scheduling.
+By default, the scheduler does not consider the architecture of a pod's container images when determining the placement of new pods onto nodes.
 
-To enable architecture-aware workload scheduling, you must create the `ClusterPodPlacementConfig` object. When you create the `ClusterPodPlacementConfig` object, this Operator deploys an operand.
+To enable architecture-aware workload scheduling, you must create the `ClusterPodPlacementConfig` object. When you create the `ClusterPodPlacementConfig` object, the Multiarch Tuning Operator deploys the necessary operands to support architecture-aware workload scheduling.
 
-When a pod is created, the operand performs the following actions: 
+When a pod is created, the operands perform the following actions: 
 
 . Add the `multiarch.openshift.io/scheduling-gate` scheduling gate that prevents the scheduling of the pod.
 . Compute a scheduling predicate that includes the supported architecture values for the `kubernetes.io/arch` label. 
 . Integrate the scheduling predicate as a `nodeAffinity` requirement in the pod specification.
 . Remove the scheduling gate from the pod.
-
-When the operand removes the scheduling gate, the pod enters the scheduling cycle. The workload is then scheduled on nodes based on the supported architectures.
-
+ 
 [IMPORTANT]
 ====
-The Technology Preview version of the Multiarch Tuning Operator does not support clusters with restricted network scenarios.
+Note the following operand behaviors:
+
+* If the `nodeSelector` field is already configured with the `kubernetes.io/arch` label for a workload, the operand does not update the `nodeAffinity` field for that workload.
+
+* If the `nodeSelector` field is not configured with the `kubernetes.io/arch` label for a workload, the operand updates the `nodeAffinity` field for that workload. However, in that `nodeAffinity` field, the operand updates only the node selector terms that are not configured with the `kubernetes.io/arch` label.
+
+* If the `nodeName` of the pod is already set, the Multiarch Tuning Operator does not process the pod.
```

Comment:
Can we change it align with above description? like below
```suggestion
* If the `nodeName` field is already set, the Multiarch Tuning Operator does not process the pod.
```

Reply:
Yes, this makes sense! Good catch :) 
updated as per the suggestion. @lwan-wanglin 
Thanks!